content/docs/aws/storage.md

```diff
@@ -48,34 +125,85 @@ Lustre is another file system that supports `ReadWriteMany`. Once difference bet
 
 ### Deploy the Amazon FSx CSI Plugin
 
+Make sure you node group instance has right permission. Check details [here](https://github.com/kubernetes-sigs/aws-fsx-csi-driver#set-up-driver-permission)
+
 ```shell
-cd ${KF_DIR}/ks_app
-export COMPONENT=aws-fsx-csi-driver
-ks generate aws-fsx-csi-driver ${COMPONENT}
-ks apply default -c ${COMPONENT}
+git clone https://github.com/kubeflow/manifests
+cd manifest/aws
+kubectl apply -k aws-fsx-csi-driver/base
 ```
 
 ### Static Provisioning
 
-You can statically provision Amazon FSx for Lustre and then pass the file system ID and DNS name to generate persistent volumes and persistent volume claims. It will create default storage class `fsx-default`.
+You can statically provision Amazon FSx for Lustre and then pass the file system ID, DNS name and mount name, to generate persistent volumes and persistent volume claims.
 
-```shell
-cd ${KF_DIR}/ks_app
-export COMPONENT=fsx-static-storage
+You can choose deployment type based on your needs. Check more details [here](https://docs.aws.amazon.com/fsx/latest/LustreGuide/using-fsx-lustre.html).
+
+<img src="/docs/images/aws/fsx-create.png"
+  alt="Amazon FSX Create Volume"
+  class="mt-3 mb-3 border border-info rounded">
 
-ks generate aws-fsx-pv-static ${COMPONENT} --fsxId=fs-048xxxx7c25 --dnsName=fs-048xxxx7c25.fsx.us-west-2.amazonaws.com
+FSX only supports single availability zone. Choose the right subnet based on your node group setting.
 
-ks apply default -c ${COMPONENT}
+<img src="/docs/images/aws/fsx-network.png"
+  alt="Amazon FSX Network Settings"
+  class="mt-3 mb-3 border border-info rounded">
+
+After you finish creating a FSX for Lustre file syste, you will have everything you need to create Persistent Volumes and Persistent Volume Claim. `File System ID`, `DNS Name` and `Mount Name` will be used later.
+
+<img src="/docs/images/aws/fsx-assets.png"
+  alt="Amazon FSX Network Settings"
+  class="mt-3 mb-3 border border-info rounded">
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: fsx-pv
+spec:
+  capacity:
+    storage: 1200Gi
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteMany
+  mountOptions:
+    - flock
+  persistentVolumeReclaimPolicy: Recycle
+  csi:
+    driver: fsx.csi.aws.com
+    volumeHandle: <your_fsx_volume_handle>
+    volumeAttributes:
```

Comment:
For some reason, this line and the `mountName` line have italic, gold formatting on the rendered page. It may be worth removing the `yaml` code-formatting marker? I'm not too worried by this, so I'll leave it up to you to decide.

Reply:
I remove `yaml` and try to see rendered page after that.

Reply:
It looks good after removing `yaml`.